src/CattyTests/DataModel/UserDataContainerTest.swift

```diff
@@ -311,8 +312,8 @@ final class UserDataContainerTest: XCTestCase {
         let container1 = UserDataContainer()
         let container2 = UserDataContainer()
 
-        container1.addObjectList(list, for: objectA)
-        container2.addObjectList(list, for: objectB)
+        container1.addList(list)
```

Comment:
This test is not of use anymore. Please remove `testIsEqualToVariablesContainerForSameNameDifferentObject`.